### PR TITLE
Adds Flipstarter/Anyonecanpay Contributions

### PR DESCRIPTION
**Pull request by SahidMiller**

### CODE REVIEW

**Review by p0o — CHANGES_REQUESTED** [review]
Thanks, your approach is very intelligent and thoughtful on working with code base's conventions. I just had some thoughts about some minor issues, please take a look. I will try to run it on my local machine by tonight as well.

File: packages/wallet/src/utils/blockchain.js [hunk 1]
```diff
@@ -77,29 +78,62 @@ export async function sendRawTx(txHex) {
...
   console.log("SLP Utxos => ", slpUtxos);
+  console.log("Frozen Utxos => ", frozenUtxos);
   console.log("SLP Balances =>", slpBalances);
 
-  // remove SLP Utxos from normal utxos
-  utxos = utxos.filter(
-    (u) => !slpUtxos.some((su) => su.txid === u.txid && su.vout === u.vout)
-  );
+  // track which frozen coins are not found in the utxo set by removing from this copy
+  const frozenCoinsNotFound = JSON.parse(JSON.stringify(frozenUtxos))
```
[the remark this answers is not in the document]
Comment (SahidMiller) [reply packages/wallet/src/utils/blockchain.js:98]: Just to be clear, the JSON.parse/stringify is just to make a quick deep copy of `frozenUtxos` that's returned from `getFrozenUtxos`. 

`getFrozenUtxos` is a wallet.js util function that always returns a JSON object even if there's an error fetching with localforage. I was modelling it using the wallet seed persistence code (which IIRC is the only other locally saved info) and so I wanted this new function to be similarly safe for calling anywhere in the codebase.

File: packages/wallet/src/config.js [hunk 2]
```diff
@@ -4,6 +4,10 @@ export const DUST = 546;
 
 export const isDevEnv = process.env.NODE_ENV === "development";
 
+const defaultHdPath = "m/44'/0'/0'/0/0"
```
[the remark this answers is not in the document]
Comment (SahidMiller) [reply packages/wallet/src/config.js:7]: Np. Thanks for teaching me to use babel for passing environment variable configuration!

File: packages/provider/src/provider.js [hunk 3]
```diff
@@ -644,6 +682,7 @@ export function cash(params = {}) {
     sendSlp,
     genesisNFTGroup,
     genesisNFTChild,
-    utility,
+    contribute,
```
[the remark this answers is not in the document]
Comment (SahidMiller) [reply packages/provider/src/provider.js]: I went with `sendAnyoneCanPayCommitment`

File: packages/provider/src/provider.js [hunk 4]
```diff
@@ -489,6 +494,39 @@ function sign(data) {
     });
 }
 
+function contribute(amount, unit, data, recipients, skipPopup = true) {
+  showRootDiv(skipPopup)
```
[the remark this answers is not in the document]
Comment (SahidMiller) [reply packages/provider/src/provider.js]: IIRC I wanted the to have the option to show the root div or not. no biggie though 👍🏿

**Pull request merged**